content/cloud-docs/run/states.mdx

```diff
@@ -31,14 +35,14 @@ _States in this stage:_
 
 _Leaving this stage:_
 
-- If the `terraform plan` command failed, the run skips to completion (**Plan Errored** state).
-- If a user canceled the plan by pressing the "Cancel Run" button, the run skips to completion (**Canceled** state).
-- If the plan succeeded and neither cost estimation nor Sentinel policy checks will be done, the run skips to completion (**Planned** state).
+- If the `terraform plan` command failed, the run does not continue (**Plan Errored** state).
+- If a user canceled the plan by pressing the "Cancel Run" button, the run does not continue (**Canceled** state).
+- If the plan succeeded with no changes and neither cost estimation nor Sentinel policy checks will be done, the run does not continue (**Planned** state).
```

Comment:
```suggestion
- If the plan succeeded with no changes and neither cost estimation nor Sentinel policy checks will be done, Terraform completes the plan and the run does not continue (**Planned** state).
```

Reply:
Hmmm, I think I see what you're getting at here, but saying "If the plan succeeded... Terraform completes the plan" seems a little confusing! Maybe something more like "Terraform Cloud considers the run complete"?